genesets/api/resources.py

```diff
@@ -230,9 +230,11 @@ def add_invite(self, request, **kwargs):
                 other_user = None
                 print("USER DIDN'T EXIST") #TODO: EMAIL USER
             if other_user is not None:
-                collaboration, created = Collaboration.objects.get_or_create(from_user=request.user,
-                                                                             to_user=other_user)
-        return self.get_detail(request, pk=request.user.pk)
+                collaboration, created = Collaboration.objects.get_or_create(
+                    from_user=request.user, to_user=other_user
+                )
+
+        return self.get_detail(request, id=request.user.pk)
```

Comment:
`get_detail()` will call `cache_ob_get()` method, which will call `obj_get()`, which will call `build_filters()` and `apply_filters()` methods.